### PR TITLE
Faiq add scratch push

### LINKED ISSUE #50: write regression test for scratch n push
Make sure we test for the fixes in #45

### PR DESCRIPTION
**Pull request by faiq**
adds tests for scratch push by creating a dummy container with special tags and grepping for those tags from docker images

fixes #50

### CODE REVIEW

File: test-all.sh [hunk 1]
```diff
@@ -63,6 +63,24 @@ testDirectMount() {
   fi
 }
 
+testScratchPush () {
+  echo -n "testing scratchnPush.."
+  testDir=$testsDir/scratch-n-push
```
Comment (hatchan) [inline test-all.sh]: The `testDirectMount` above uses all lower case, this one still uses camel case.

File: tests/projects/scratch-n-push/wercker.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+build:
+  box:
+    id: busybox
+    cmd: /bin/sh
+  steps:
+    - internal/docker-push:
+        tag: uniqueTagFromTest, alsouniqueTagFromTest
```
Comment (hatchan) [inline tests/projects/scratch-n-push/wercker.yml]: If we're doing two tags, then we should also have 2 tests

**Comment by hatchan** [comment]
:+1:

File: tests/projects/scratch-n-push/wercker.yml (hunk 2 above)
Comment (faiq) [inline tests/projects/scratch-n-push/wercker.yml]: by two tests do you mean two grep tests?

File: test-all.sh (hunk 1 above)
Comment (faiq) [inline test-all.sh]: whoops must have been a vim mistake! `testDirectMount` should be camelCase

**Comment by termie** [comment]
looks bueno :)

**Pull request merged**